WordPress/Classes/Services/MediaThumbnailCoordinator.swift

```diff
@@ -80,13 +78,11 @@ class MediaThumbnailCoordinator: NSObject {
             return
         }
 
-        coreDataStack.performAndSave { context in
-            let mediaService = MediaService(managedObjectContext: context)
-            mediaService.getMediaWithID(mediaID, in: media.blog, success: { (loadedMedia) in
-                onCompletion(loadedMedia, nil)
-            }, failure: { (error) in
-                onCompletion(nil, error)
-            })
-        }
+        let mediaService = MediaService(managedObjectContext: coreDataStack.mainContext)
+        mediaService.getMediaWithID(mediaID, in: media.blog, success: { (loadedMedia) in
+            onCompletion(loadedMedia, nil)
```

Comment:
What do you think of tracking what your wrote in [the comment above](https://github.com/wordpress-mobile/WordPress-iOS/pull/20294/files#r1131572886) in the code?

```suggestion
        // It's only safe to use the main context as the MediaService instance's context because a Media object is 
        // leaked out of MediaService's lifecycle (MediaService.managedObjectContext's to be exact.
        let mediaService = MediaService(managedObjectContext: coreDataStack.mainContext)
        mediaService.getMediaWithID(mediaID, in: media.blog, success: { (loadedMedia) in
            onCompletion(loadedMedia, nil)
```

Reply:
Also, SwiftLint might at some point complain about the redundant `()` around `loadedMedia`. 🤷‍♂️